f2/.storybook/themeDecorator.js

```diff
@@ -7,6 +7,8 @@ import { I18nProvider } from '@lingui/react'
 import { i18n } from '@lingui/core'
 import { activate } from '../src/i18n.config'
 import { StateProvider, initialState, reducer } from '../src/utils/state'
+import '../src/theme/style.css'
```

Comment:
Can we reverse the order of these imports. This way our custom styles will take precedence over the default Bootstrap style.